app/admin/ministry.rb

```diff
@@ -34,7 +33,7 @@
     f.semantic_errors
     f.inputs do
       f.input :name
-      f.input :description, as: :ckeditor
```

Comment:
The CK Editor is still used for editing other model attributes, like `Course#description` and `Conference#description`.